draft-ietf-mls-protocol.md

```diff
@@ -1611,6 +1623,11 @@ struct {
           Commit commit;
           opaque confirmation_tag<0..255>;
     }
+} MLSPlaintextContent;
+
+struct {
+    MLSPlaintextSignatureContext context;
+    MLSPlaintextTBS tbs;
 } MLSPlaintextTBS;
```

Comment:
Looks like something went wrong here. The new `MLSPlaintextContext` and `MLSPlaintextContent` structs are not used and the `MLSPlaintextTBS` struct is now self-referencing.

If I had to guess, I think you meant:

```suggestion
struct {
    MLSPlaintextContext context;
    MLSPlaintextContent content;
} MLSPlaintextTBS;
```